src/state.rs

```diff
@@ -33,7 +33,13 @@ pub struct CallFrame {
     pub storage: Rc<RefCell<dyn Storage>>,
     /// Transient storage should be used for temporary storage within a transaction and then discarded.
     pub transient_storage: InMemory,
+    pub context_u128: u128,
+    pub this_address: Address,
+    pub msg_sender: Address,
+    pub code_address: Address,
+    pub ergs_remaining: u32,
```

Comment:
Maybe add all the new values in a new struct in order to make it cleaner?

Reply:
[This fields are now part of the `CallFrame` struct](https://github.com/lambdaclass/era_vm/blob/2562a1f2d100f1fbb3d7debe5a9c487ff5930858/src/call_frame.rs#L29-L36), I cant try moving them into a 
new `Context` struct so that `CallFrame` doesn't get saturated.

Reply:
Try to name it something else than `Context` since Near Call already has a `Context` Struct, so we don't overlap later

Reply:
We can choose later which of the 2 should actually be called `Context`

Reply:
Looking at the [`Context` struct](https://github.com/lambdaclass/era_vm/pull/48/files#diff-f45de6772e2dbec81333a31f1f67583428ef2353e386df23dbbe69ba83a48c37R29-R33) being introduced in #48, I'm wondering if it wouldn't be more suitable to add these new fields that struct considering that it's share beetwen the `frame` and `near_call_frames`.

Am I'm missing something?

Reply:
It depends, do these new field change when calling a contract with near or far call? If they don't, then its fine to put them on `Context`, if they do, they belong on `CallFrame`

Reply:
Marking as resolved ([actually continued here](https://github.com/lambdaclass/era_vm/pull/44#discussion_r1669043582))